include/ibamr/StaggeredStokesSolver.h

```diff
@@ -67,11 +68,6 @@ class StaggeredStokesSolver : public virtual IBTK::GeneralSolver
      */
     StaggeredStokesSolver();
 
-    /*!
-     * \brief Destructor.
-     */
-    ~StaggeredStokesSolver();
```

Comment:
I think we've adopted the ` = default` syntax instead of removing the definition completely.